docs/book/pattern/output-cache.md

```diff
@@ -5,30 +5,54 @@ The `OutputCache` pattern caches output between calls to `start()` and `end()`.
 ## Quick Start
 
 ```php
-use Laminas\Cache\PatternFactory;
+use Laminas\Cache\Pattern\OutputCache;
+use Laminas\Cache\Pattern\PatternOptions;
 
-$outputCache = PatternFactory::factory('output', [
-    'storage' => 'apc'
-]);
+$outputCache = new OutputCache(
+    $storage,
+    new PatternOptions()
+);
 ```
 
+> ### Storage Adapter
+>
+> The `$storage` adapter can be any adapter which implements the `StorageInterface`. Check out the [Pattern Quick Start](./intro.md#quick-start)-Section for a standard adapter which can be used here.
+
 ## Configuration Options
 
 Option | Data Type | Default Value | Description
 ------ | --------- | ------------- | -----------
-`storage` | `string | array | Laminas\Cache\Storage\StorageInterface` | none | Adapter used for reading and writing cached data.
+`storage` | `string\|array\|Laminas\Cache\Storage\StorageInterface` | none | **deprecated** Adapter used for reading and writing cached data.
+
+## Examples
+
+### Caching simple View Scripts
+
+```php
+use Laminas\Cache\Pattern\OutputCache;
+use Laminas\Cache\Pattern\PatternOptions;
+
+$outputCache = new OutputCache(
+    $storage,
+    new PatternOptions()
+);
+
+$outputCache->start('mySimpleViewScript');
+include '/path/to/view/script.phtml';
+$outputCache->end();
+```
```

Comment:
Avoid words like "simple".

```suggestion
### Caching a View Scripts

```php
use Laminas\Cache\Pattern\OutputCache;
use Laminas\Cache\Pattern\PatternOptions;

$outputCache = new OutputCache(
    $storage,
    new PatternOptions()
);

$outputCache->start('myViewScript');
include '/path/to/view/script.phtml';
$outputCache->end();
```
```

Reply:
Fixed, most of the documentation already existed. Not quite sure why all of them have changes.